analyzere/base_resources.py

```diff
@@ -148,6 +148,12 @@ def to_dict(self):
             d[k] = to_dict(v)
         return d
 
+    @classmethod
+    def from_dict(cls, d):
```

Comment:
I think you can just do `AnalyzeReObject(**my_dict)` to construct one from a dict; was that not working?